tests/framework/ROM/TimeSeries/MultiResolutionTSA/gold/MultiYearDWT/samples.csv

```diff
@@ -0,0 +1,4001 @@
+RAVEN_sample_ID,seconds,macro,scaling,signal0,PointProbability,prefix,ProbabilityWeight
```

Comment:
Is it possible to reduce the size of this file? In addition, you do not need ProbabilityWeight, PointProbability in your output.

Reply:
I can reduce the number of levels requested per year and regold. 
I'm unsure how to get rid of ProbabilityWeight and PointProbability.. I don't request those in the raven input XML, I wonder if they're there automatically because there are multiple target signals?